### PR TITLE
Issue 186780 add action input

### PR DESCRIPTION
**Pull request by jdotcms**
This simple PR, is adding a bit more info into a WorkflowAction by wrapping it into a WorkflowActionView
The idea will be to have the inputs needed by angular for the actions (for instance when need to popup the PP dialog, comment, etc)

There is a body in there, but it will be for future usages

### CODE REVIEW

**Review by dsilvam — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+/**
+ * This class encapsulates the WF action inputs
+ * this input will be taken by the FE and show some input (such as dialog)
+ * There are a few known dialogs such as PP, Assign or Comment that are included on dotCMS
+ * But the user might describe a set of inputs into the body in order to create a form input
+ * @author jsanca
+ */
+public class ActionInputView {
+
+    private final String id;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java:14]: Codacy found an issue: [Avoid variables with short names like id](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5896898)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private final Map<String, Object> body;
+
+    public ActionInputView(final String id, final Map<String, Object> body) {
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java:17]: Codacy found an issue: [Avoid variables with short names like id](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5896898)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java:14]: Codacy found an issue: [Avoid variables with short names like id](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5896898)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/ActionInputView.java:17]: Codacy found an issue: [Avoid variables with short names like id](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5896898)

**Pull request merged**